0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -213,6 +214,13 @@ Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promis
 
 **TODO**: Define what the message format is.  Plugin must be connected, otherwise the promise should reject.
 
+#### rejectIncomingTransfer
+<code>ledgerPlugin.rejectIncomingTransfer( **transferId**:String ) ⇒ Promise.&lt;null></code>
```

Comment:
Added the `rejectMessage`.

> What use cases are there for replyToTransfer?

Originally it was intended for sending error messages back. It might be useful in the case of Optimistic payments to notify the sender whether the transfer got through. I can't think of anything else right now though.

> Is there a use case for returning a message with success (fulfill)?

The only thing I could think of would be something like "I did this one, but you (sender) should do something different next time". Not a very important use case. One way to look at it would be "it's finished, we don't need to talk about it anymore". If we want to add something like this it should be done in a new PR.